homeassistant/components/route53/__init__.py

```diff
@@ -56,7 +57,7 @@ def update_records_interval(now):
             aws_access_key_id, aws_secret_access_key, zone, domain, records, ttl
         )
 
-    def update_records_service(now):
+    def update_records_service(call: ServiceCall) -> None:
```

Comment:
The argument isn't used, and I think the old name `now` was probably a bad copy paste.